### PR TITLE
Added goerli test cases of arka and skandha

### PR DESCRIPTION
**Pull request by Jineshdarjee**
## Description
* Added Arka and Skandha test cases for Goerli Network.
* Modified and added scripts in Package.json.
* Added test data in the test data files.
* Removed swap test cases for testnet, During the testnet execution it'll take mainnet's swap test cases.

### CODE REVIEW

**Comment by Jineshdarjee** [comment]
Anyone, please review the PR.

File: test/specs/testnet/01_TransferringFunds/Goerli.spec.js
Comment (IAmKio) [inline test/specs/testnet/01_TransferringFunds/Goerli.spec.js:1]: There's a lot of commented code here, is that dead code? If so, please remove it

File: test/specs/testnet/01_TransferringFunds/Mumbai.spec.js
Comment (IAmKio) [inline test/specs/testnet/01_TransferringFunds/Mumbai.spec.js:1]: As above

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few things:

- Please remove any dead code
- Upon inspection, it looks like you've just commented out the code because it's not currently needed right now. If this is the code, would it not be better to use a condition to execute the code?
- The two files with nearly everything removed, should you not just delete the files themselves or are they still needed for a reason?

File: test/specs/testnet/01_TransferringFunds/Goerli.spec.js
Comment (Jineshdarjee) [reply test/specs/testnet/01_TransferringFunds/Goerli.spec.js:1]: I have removed the command code.

File: test/specs/testnet/01_TransferringFunds/Mumbai.spec.js
Comment (Jineshdarjee) [reply test/specs/testnet/01_TransferringFunds/Mumbai.spec.js:1]: I have removed the command code.

**Comment by Jineshdarjee** [comment]
Updated code I'll push in the next PR. 
@IAmKio Can I close this PR?

**Comment by IAmKio** [comment]
> Updated code I'll push in the next PR. @IAmKio Can I close this PR?

Sorry can i just ask why this PR is being closed and what is happening in the next PR?

**Comment by Jineshdarjee** [comment]
In the new PR, I have added new test cases and Also I have removed all the commented code as mentioned in this PR.

**Comment by Jineshdarjee** [comment]
1. Added smoke and regression test cases of the Exchange Offer. 
2. Added smoke and regression test cases of the Paymaster. (mainnet and testnet)
3. Make the retry count dynamic in all the test cases.
4. Removed a couple of validations from the NFTList test case.
5. Added smoke and regression test cases of the Token Lists. (mainnet and testnet)
6. Moved smoke and regression test cases of the NFT List. (mainnet and testnet)
7. Added smoke and regression test cases of the Exchange Rates. (mainnet and testnet)
8. Added smoke test cases of the Get ZeroDev Address and Get SimpleAccount Address. (mainnet and testnet)
9. Added smoke and regression test cases of the Tranfer Token with Arka Paymaster. (mainnet and testnet)
10. Added smoke and regression test cases of the Tranfer Token with Arka Pimlico Paymaster. (mainnet and testnet)
11. Removed the commented code as mentioned by Kio in the previous PR.
12. Only xdai network is enabled in e2e execution.

**Comment by Jineshdarjee** [comment]
## Description
1. Added test cases of the Arka Paymaster with validuntil and validafter. (mainnet and testnet)
2. Remove UserOpReceipt code, To decrease the execution time.

**Comment by Jineshdarjee** [comment]
## Description
1. Modified the script for null value of the UserOpsReceipts

**Comment by Jineshdarjee** [comment]
## Description
1. Disabled Arka Pimlico paymaster test cases for xdai network.

**Review by IAmKio — APPROVED** [review]
This PR is too large to review - i've seen most of it and can see the pattern. We need to try and make any future PRs smaller and more feature focused, i'm unsure how they are this large to begin with.

**Pull request merged**